src/main/java/org/asamk/signal/http/HttpServerHandler.java

```diff
@@ -0,0 +1,123 @@
+package org.asamk.signal.http;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpServer;
+
+import org.asamk.signal.commands.Commands;
+import org.asamk.signal.jsonrpc.JsonRpcReader;
+import org.asamk.signal.jsonrpc.JsonRpcResponse;
+import org.asamk.signal.jsonrpc.JsonRpcSender;
+import org.asamk.signal.jsonrpc.SignalJsonRpcCommandHandler;
+import org.asamk.signal.manager.Manager;
+import org.asamk.signal.manager.MultiAccountManager;
+import org.asamk.signal.util.Util;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.concurrent.Executors;
+
+public class HttpServerHandler {
+
+    private final static Logger logger = LoggerFactory.getLogger(HttpServerHandler.class);
+
+    private final ObjectMapper objectMapper = Util.createJsonObjectMapper();
+
+    private final InetSocketAddress address;
+
+    private final Manager m;
+
+    private final MultiAccountManager c;
+
+    public HttpServerHandler(final InetSocketAddress address, final Manager m) {
+        this.address = address;
+        this.m = m;
+        this.c = null;
+    }
+
+    public HttpServerHandler(final InetSocketAddress address, final MultiAccountManager c) {
+        this.address = address;
+        this.m = null;
+        this.c = c;
+    }
+
+    public void init() {
+
+        try {
+
+            logger.info("Starting server on port " + address.toString());
+
+            final var server = HttpServer.create(new InetSocketAddress(address.getPort()), 0);
```

Comment:
You can just use the passed address here directly.
```suggestion
            final var server = HttpServer.create(address, 0);
```

Reply:
Oups. Fixed that.